pycon-fr-2015/category.json

```diff
@@ -1,3 +1,5 @@
 {
+  "alias": "category/73/PyCon-FR-2015",
```

Comment:
It looks like [`"alias"` is deprecated](https://github.com/pyvideo/data/blob/master/.schemas/category.json#L5) now